frontend/src/components/secure-message/MessageDisplay.vue

```diff
@@ -713,6 +732,28 @@ export default {
           this.loading = false;
         });
     },
+    removeNote(noteID) {
+      this.processing = true;
+      this.loading = true;
+
+      ApiService.apiAxios.put(`${Routes.edx.EXCHANGE_URL}/${this.secureExchangeID}/removeNote/${noteID}`)
+        .then((response) => {
+          this.getExchange();
+          if(response.status === 200){
+            this.setSuccessAlert('Success! The note has been removed.');
+          } else{
+            this.setSuccessAlert('Error! The note was not removed.');
```

Comment:
should this be an errorAlert? I would actually move this to the .catch statement and you don't need to check the response.status. So something like 
. 
            this.setSuccessAlert('Success! The note has been removed.');
          this.closeNoteIndex();
        })
        .catch(error => {
            this.setErrorAlert('Error! The note was not removed.');
          console.log(error);
        })

Reply:
I'm pretty sure if the call gets past the .then(), we know it's successful.